components/timezone/src/ids.rs

```diff
@@ -333,6 +333,19 @@ impl TimeZoneIdMapperBorrowed<'_> {
         Some(string)
     }
 
+    /// Iterates through all canonical time zones
+    pub fn iter_canonical(&self) -> impl Iterator<Item = TimeZoneBcp47Id> + '_ {
+        self.data.bcp47_ids.iter()
+    }
+
+    /// Iterates through all metazones
+    pub fn iter_metazones(&self) -> impl Iterator<Item = TimeZoneBcp47Id> + '_ {
+        self.data
+            .metazones
+            .iter()
+            .filter_map(|i| self.data.bcp47_ids.get(i as usize))
+    }
```

Comment:
Issue: huh? The function returns an iterator of `TimeZoneBcp47Id` but it is documented as returning an iterator of metazones.